tests/test_dem_io.py

```diff
@@ -72,7 +72,7 @@ def d3d(tmpdir, dem_source, kwargs):
     # get dem
     df = pdem.dem(dem_source=dem_source, **kwargs)
     # get dem on grid
```

Comment:
`s/grid/mesh/g` (or even remove the comment. it just repeats the code, it doesn't clarify anything)